src/test/datascience/notebook/helper.ts

```diff
@@ -699,7 +695,6 @@ async function selectPythonRemoteKernelConnectionForActiveInterpreter(
 export async function waitForKernelToGetAutoSelected(
     notebookEditor: NotebookEditor,
     expectedLanguage: string,
-    preferRemoteKernelSpec: boolean = false,
```

Comment:
There's no need to specify this value, it can be derived from `IS_REMOTE_NATIVE_TEST`